specs/dwritecore/DWriteCore.md

```diff
@@ -208,10 +908,12 @@ enum DWRITE_FACTORY_TYPE
     // Warning: Make sure any additional new enum values are consistent between different versions of
     // this header in DWriteCore and the Windows SDK.
     DWRITE_FACTORY_TYPE_ISOLATED2
```

Comment:
>DWRITE_FACTORY_TYPE_ISOLATED2 [](http://example.com/codeflow?start=4&length=29)

`ISOLATED2` doesn't tell me much about its behavior. Some options:

- `DWRITE_FACTORY_TYPE_ISOLATED_TO_PROCESS`
- `DWRITE_FACTORY_TYPE_ISOLATED_NO_CROSS_PROCESS_CACHE`
- `DWRITE_FACTORY_TYPE_RESTRICTED`

Using a number suffix for the interfaces makes sense to me, but we should be able to be a little more descriptive with the enumerant name.

For the record, this should not be considered a comprehensive list of suggestions -- I'm just hoping we can find a more descriptive name.

Reply:
This enum value was added in February as part of https://github.com/microsoft/WindowsAppSDK/pull/198. I proposed the name DWRITE_FACTORY_TYPE_RESTRICTED, but Raymond Chen and other reviewers thought that was confusing and suggested ISOLATED2. You can see the comments in the original PR linked above.